### PR TITLE
MEN-4382: Add deviceconfig service to the integration components

### PR DESCRIPTION
**Pull request by alfrunes**

### CODE REVIEW

**Review by lluiscampos — COMMENTED** [review]
Please do not merge this until the traefik PRs + mender-docs + the release is done :)

File: component-maps.yml
Comment (kacf) [inline component-maps.yml:502]: You are missing an entry under the `docker_image` top level key. These blocks always come in threes.

**Review by kacf — CHANGES_REQUESTED** [review]

File: docker-compose.config.yml
Comment (lluiscampos) [inline docker-compose.config.yml:5]: The tag here should be `mender-master`

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Looks good to me otherwise.

File: docker-compose.config.yml
Comment (mchalski) [inline docker-compose.config.yml:14]: I'm here mostly for this section, +1 :)

**Review by lluiscampos — APPROVED** [review]

**Comment by lluiscampos** [comment]
> Please do not merge this until the traefik PRs + mender-docs + the release is done :)

I won't block this any-more. Just coordinate with @kjaskiewiczz and the `mender-docs` test fixes.

**Comment by alfrunes** [comment]
The [QA pipeline](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/246993880) with mendersoftware/mender-qa#477 passed.
@kacf I'm awaiting your second round of review before I'm merging all the deviceconfig integration PRs.

**Review by kacf — APPROVED** [review]
Perfect! Sorry for the delay!

**Pull request merged**
